src/modules/Adapter/index.js

```diff
@@ -358,6 +358,11 @@ export default class Adapter extends AdapterModuleCore {
         });
         break;
       }
+      case '/custom-alert-message':
+        if (data.alertMessage) {
+          this._alert.warning({ message: `showCustomAlertMessage:${data.alertMessage}` });
```

Comment:
We can use `this._alert.alert({ level: 'warning', message: '' })`. And we can also support to set message level